config.json

```diff
@@ -9,63 +9,41 @@
+    "params": [
+      {
+        "name": "accessKeyId",
+        "viewName": "Access Key ID",
+        "type": "vault",
+        "description": "The default AWS Access Key ID to use for authentication.",
+        "placeholder": "AKIAIOSFODNN7EXAMPLE",
+        "learnUrl": "https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey"
+      },
+      {
+        "name": "secretAccessKey",
+        "viewName": "Access Key Secret",
+        "type": "vault",
+        "description": "The default AWS Access Key secret to use for authentication.",
+        "placeholder": "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
+        "learnUrl": "https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey"
+      }
+    ]
+  },
   "settings": [
-    {
-      "name": "accessKeyId",
-      "viewName": "Access key",
-      "type": "vault",
-      "description": "The default Access Key ID to use to authenticate to AWS.",
-      "placeholder": "AKIAIOSFODNN7EXAMPLE",
-      "learnUrl": "https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey"
-    },
-    {
-      "name": "secretAccessKey",
-      "viewName": "Secret key",
-      "type": "vault",
-      "description": "The default Secret Key to use to authenticate to AWS.",
-      "placeholder": "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
-      "learnUrl": "https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey"
-    },
     {
       "name": "region",
       "viewName": "Region",
       "type": "string",
-      "description": "The default AWS region to make requests on.",
-      "placeholder": "us-east-2",
-      "learnUrl": "https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/using-regions-availability-zones.html"
+      "description": "The ID of the default region to make requests on.",
+      "learnUrl": "https://docs.aws.amazon.com/general/latest/gr/s3.html"
     }
   ],
   "methods": [
     {
       "name": "createApplication",
       "viewName": "Create EC2 Application",
       "params": [
-        {
-          "name": "accessKeyId",
-          "viewName": "Access key",
-          "type": "vault",
-          "description": "The Access Key ID to use to authenticate to AWS for this request.",
-          "placeholder": "AKIAIOSFODNN7EXAMPLE",
-          "learnUrl": "https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey"
-        },
-        {
-          "name": "secretAccessKey",
-          "viewName": "Secret key",
-          "type": "vault",
-          "description": "The Secret Key to use to authenticate to AWS for this request.",
-          "placeholder": "wJalrXUtnFEMI/K7MDENG/bPxRfiCYEXAMPLEKEY",
-          "learnUrl": "https://docs.aws.amazon.com/IAM/latest/UserGuide/id_credentials_access-keys.html#Using_CreateAccessKey"
-        },
-        {
-          "name": "region",
```

Comment:
The user might want to edit region per action. Having this in settings only, disallows the user from using two regions at separate pipelines.